doc/source/_static/custom.css

```diff
@@ -20,28 +20,32 @@
     box-shadow: 0 0rem 0rem 0 rgb(255,172,0); /* var(--pst-color-shadow); */
 }
 
+/* Navbar elements */
+.bd-navbar-elements {
+    text-align: center;
+}
+
+/* Centering the page content */
+.bd-main .bd-content {
+    display: flex;
+    justify-content: center;
+}
+
+#pst-back-to-top {
+    background: 
+}
+/* Removing navbar shadow */
+.sd-shadow-sm {
+    box-shadow: 0 0rem 0rem rgba(250, 250, 250, 0.6) !important;
```

Comment:
```suggestion
    box-shadow: 0 0rem 0rem rgba(250, 250, 250, 0.6) !important;
```
i am not very sure about this, because .sd is the sphinx design class which has no effect with navigation bar.

Reply:
for me it works.... there is no shadow between the navbar and the content.

<img width="1749" alt="image" src="https://github.com/user-attachments/assets/9c3a8dd6-6a6e-44f6-966f-5543dfd89ec8" />